blogs/whats_new_1.6/index.md

```diff
@@ -0,0 +1,231 @@
+# What's new in Network Observability 1.6
+
+Network Observability 1.6 was released on June 17, 2024.  Even though this is considered a minor version upgrade from 1.5, it is a significant release that could lower the barrier to adoption into production.
+
+But before we go further, for those of you new to Network Observability, NetObserv, for short, is an optional operator that provides a slew of capabilities to track and provide insight into your network traffic flows.  While it works on any Kubernetes cluster, it works even better in an OpenShift environment, which is what I will focus on in this article.  I will only discuss the new features in this release so if you want the full feature list, read the documentation on [About Network Observability](https://docs.openshift.com/container-platform/4.16/observability/network_observability/network-observability-overview.html).
```

Comment:
should we still say that netobserv is an operator or start saying that it's a set of tools ? :upside_down_face:

Reply:
Good suggestion!

Reply:
I'm going to leave it as operator and then when Network Observability CLI becomes GA, let's revisit!